### PR TITLE
Adding a cache mechanism to dataspace #2619 : fixing tests

### PR DESCRIPTION
**Pull request by fviale**
- trying to fix TestCacheSpaceCleaning
- added node log4j configuration to rest tests to have more debug output

### CODE REVIEW

File: rm/rm-node/src/main/java/org/ow2/proactive/resourcemanager/nodesource/dataspace/DataSpaceNodeConfigurationAgent.java [hunk 1]
```diff
@@ -304,6 +304,7 @@ public void run() {
                     FileObject[] files = rootFO.findFiles(Selectors.EXCLUDE_SELF);
                     for (FileObject file : files) {
                         if (currentTime - file.getContent().getLastModifiedTime() > invalidationPeriod) {
```
Comment (sonarqube-activeeon) [inline rm/rm-node/src/main/java/org/ow2/proactive/resourcemanager/nodesource/dataspace/DataSpaceNodeConfigurationAgent.java]: MAJOR Refactor this code to not nest more than 3 if/for/while/switch/try statements. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS134)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by fviale** [comment]
Retest this please

**Comment by paraita** [comment]
Retest this please

**Comment by paraita** [comment]
Retest this please

**Comment by marcocast** [comment]
retest this please

**Pull request closed without merging**